src/tab.rs

```diff
@@ -26,6 +26,8 @@ pub fn render_tab(text: String, tab: &TabInfo, palette: Palette, separator: &str
         } else {
             palette.green
         }
+    } else if alternate_color {
+        palette.red
```

Comment:
TODO: if pipe message includes the exit code maybe this could be either red/green?